working-docs/slack-channel-introduction.md

```diff
@@ -0,0 +1,44 @@
+
+# New Here? Walkthrough
+New to the Diversity & Inclusion slack channel?
+
+Here’s a couple of things you should know before you dive into the channel!
+
+## What is DD&I
+
+Drupal Diversity and Inclusion is an independent collective of volunteers who advocate for greater diversity in the community and more inclusive events, help create safe spaces, and build the capacity of allies.
+
+This group was created after DrupalCon New Orleans. Nikki Stevens & Karyn Cassio gave a [presentation about diversity and inclusion](https://events.drupal.org/neworleans2016/sessions/together-we-can-make-drupal-example-diversity-tech "presentation about diversity and inclusion").
+
+Nikki facilitated a follow up BOF was held the next day. Participants had so many ideas, and concerns and desires that we couldn’t do justice to most of them in the short amount of time we had. The general consensus seemed to be “Let’s keep talking about this so we can address everyone’s thoughts.” and so this group was created.
+
+## Who we are
+
+The DD&I team does not yet have a formal structure. We are a collective of volunteers who all believe in advocating for diversity and inclusion.
+
+We are not affiliated with the DA and our booth/stickers are all funded by donations.
+
+## What this channel is for
+
+Through this channel, we aim to to continue the conversation about diversity and inclusion within Drupal by providing a venue for important and sometimes difficult conversations that we need to have as a community.
+
+We would like to:
+- provide a safe (but moderated?) space to discuss issues of diversity and inclusion within the Drupal community.
+- provide support for people who are marginalized
+- advocate for a more open and diverse community whether through these conversations, presentations at camps and conferences, and BOF sessions.
```

Comment:
- Advocate for a more open and diverse Drupal community through Slack/IRC conversations, presentations and camps and conferences, BOF sessions, and other means

I started out by eliminating "whether" (which is superfluous), and got carried away. :)

Also, for consistency, all bullet items should start with a capital letter, and all should ether have or not have a period (or other appropriate punctuation) at the end.